services/src/main/java/io/druid/cli/PullDependencies.java

```diff
@@ -94,6 +93,14 @@
       // Different tasks which are classloader sensitive attempt to maintain a sane order for loading libraries in the
       // classloader, but it is always possible that something didn't load in the right order. Also we don't want to be
       // throwing around a ton of jars we don't need to.
+      //
+      // Here is a list of dependencies extensions should probably exclude.
+      //
+      // Conflicts can be discovered using the following command on the distribution tarball:
+      //    `find lib -iname *.jar | cut -d / -f 2 | sed -e 's/-[0-9]\.[0-9]/@/' | cut -f 1 -d @ | sort | uniq | xargs -I {} find extensions -name "*{}*.jar" | sort`
```

Comment:
probably this should be moved to docs for extending druid? 
http://druid.io/docs/latest/development/modules.html